### PR TITLE
Subclass WGL and move theme code

### PR DESCRIPTION
**Pull request by Meakk**
Subclass `vtkWin32OpenGLRenderWindow` and move all Windows related code (dark theme) there.

### CODE REVIEW

File: vtkext/private/module/vtkF3DWGLRenderWindow.cxx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#include <Windows.h>
+#include <dwmapi.h>
+
+#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
+#define DWMWA_USE_IMMERSIVE_DARK_MODE 20
+#endif
+
+constexpr auto IMMERSIVE_DARK_MODE_SUPPORTED_SINCE = 19041;
+
+vtkStandardNewMacro(vtkF3DWGLRenderWindow);
```
Comment (mwestphal) [inline vtkext/private/module/vtkF3DWGLRenderWindow.cxx:14]: standard new macro are usually put after internal unamed namespace

**Comment by mwestphal** [comment]
There is a conflict to fix

File: vtkext/private/module/CMakeLists.txt [hunk 2]
```diff
@@ -61,8 +61,8 @@ if(NOT VTK_VERSION VERSION_GREATER_EQUAL 9.2.20220907)
   set(classes ${classes} vtkF3DOrientationMarkerWidget)
 endif()
 
-if(WIN32 AND F3D_WINDOWS_GUI)
```
[the remark this answers is not in the document]
Comment (Meakk) [reply vtkext/private/module/CMakeLists.txt:64]: It doesn't hurt to build it even if it's not activated by the factory.  
It simplifies the CMake logic.

File: vtkext/private/module/vtkF3DWGLRenderWindow.cxx (hunk 1 above)
Comment (Meakk) [reply vtkext/private/module/vtkF3DWGLRenderWindow.cxx:14]: will move it

File: vtkext/private/module/CMakeLists.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (mwestphal) [reply vtkext/private/module/CMakeLists.txt:64]: fair

**Comment by Meakk** [comment]
@mwestphal PTAL again

**Review by mwestphal — APPROVED** [review]

**Pull request merged**